elyra/pipeline/processor_kfp.py

```diff
@@ -234,6 +238,10 @@ def _cc_pipeline(self, pipeline, pipeline_name):
             pipeline_envs['AWS_SECRET_ACCESS_KEY'] = cos_password
             # Convey pipeline logging enablement to operation
             pipeline_envs['ELYRA_ENABLE_PIPELINE_INFO'] = str(self.enable_pipeline_info)
+            # Setting identifies a writable directory in the container image.
+            # Only Unix-style path spec is supported.
+            pipeline_envs['ELYRA_WRITABLE_CONTAINER_DIR'] =\
+                os.getenv('ELYRA_WRITABLE_CONTAINER_DIR', '/tmp').rstrip('/')
```

Comment:
See previous.

Reply:
I will use a [global] variable in the assignment as you suggested earlier.

Reply:
done